tracker/sm-website-update-visitor.js

```diff
@@ -1,147 +1,153 @@
+        updateVisitorCustomAttributes("videoWatched","yes")
       });
 
       $('#First-Name, #Last-Name').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
-          name = $('#First-Name').val() + " " + $('#Last-Name').val()
-          updateVisitorDetails("name", name);
+        if ($(this).val().length > 0) {
+           name = $('#First-Name').val() + " " + $('#Last-Name').val()
+           updateVisitorInformation("name", name);
         };
       });
 
       $('#First-Name-3, #Last-Name-3').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
+        if ($(this).val().length > 0 ) {
           name = $('#First-Name-3').val() + " " + $('#Last-Name-3').val()
-          updateVisitorDetails("name", name);
+          updateVisitorInformation("name", name);
         }
       });
 
       $('#First-Name-2, #Last-Name-2').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
+        if ($(this).val() > 0 ) {
           name = $('#First-Name-2').val() + " " + $('#Last-Name-2').val()
-          updateVisitorDetails("name", name);
+          updateVisitorInformation("name", name);
         }
       });
-      
+
       $('#Company-Name, #Company-Name-3, #Company-2').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
-          company = $(this).val();
-          currentCustomAttributes["companyName"] = company
-          updateVisitorDetails("custom_attributes", currentCustomAttributes);;
+        if ($(this).val() > 0) {
+          companyName = $(this).val();
+          updateVisitorCustomAttributes("companyName", companyName);
         }
       });
 
-     $('#Job-Title-2').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
-          JobPosition = $(this).val();
-          currentCustomAttributes["JobTitle"] = JobPosition
-          updateVisitorDetails("custom_attributes", currentCustomAttributes);
+      $('#Job-Title-2').change(function(){
+        if ($(this).val().length > 0 ) {
+          jobTitle = $(this).val();
+          updateVisitorCustomAttributes("jobTitle", jobTitle);
         };
       });
 
       $('#Email-2, #Email-5, #email-4, #email').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
+        if ($(this).val().length > 0 ) {
           email = $(this).val();
-          updateVisitorDetails("email", email)
+          updateVisitorInformation("email", email);
         };
       });
 
       $('#Phone-2').change(function(){
-        if ($(this).val() === "") {
-          return
-        } else {
+        if ($(this).val().length > 0) {
           phone = $(this).val();
-          updateVisitorDetails("phone", phone);
+          updateVisitorInformation("phone", phone);
         };
       });
     }
-  ); 
-  
-function getHeaders(){
```

Comment:
this function sets some global values. it does not make sense then to be function.